cumulus_etl/etl/tasks/base.py

```diff
@@ -145,7 +145,8 @@ async def _write_tables_in_batches(self, entries: EntryIterator) -> None:
                 self.table_batch_cleanup(table_index, batch_index)
 
                 print(f"  {summary.success:,} processed for {formatter.dbname}")
-                batch_index += 1
+
+            batch_index += 1
```

Comment:
Unrelated, but noticed that this counter had the wrong indenting. This didn't actually matter, unless someone was doing ndjson exports of medicationrequests.